quantum/keymap_extras/keymap_norman.h

```diff
@@ -0,0 +1,89 @@
+
+/* Copyright 2019 Torben Hoffmann
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "keymap.h"
+// For software implementation of norman
+#define NM_Q    KC_Q
+#define NM_W    KC_W
+#define NM_D    KC_E
+#define NM_F    KC_R
+#define NM_K    KC_T
+#define NM_J    KC_Y
+#define NM_U    KC_U
+#define NM_R    KC_I
+#define NM_L    KC_O
+#define NM_SCLN KC_P
+
+#define NM_A    KC_A
+#define NM_S    KC_S
+#define NM_E    KC_D
+#define NM_T    KC_F
+#define NM_G    KC_G
+#define NM_Y    KC_H
+#define NM_N    KC_J
+#define NM_I    KC_K
+#define NM_O    KC_L
+#define NM_H    KC_SCLN
+#define NM_COLN LSFT(NM_SCLN)
+
+#define NM_Z    KC_Z
+#define NM_X    KC_X
+#define NM_C    KC_C
+#define NM_V    KC_V
+#define NM_B    KC_B
+#define NM_P    KC_N
+#define NM_M    KC_M
+#define NM_COMM KC_COMM
+#define NM_DOT  KC_DOT
+#define NM_SLSH KC_SLSH
+
+// Make it easy to support these in macros
```

Comment:
I believe what you actually need here is a `sendstring_norman.h` file to pair with `keymap_norman.h`.

This file should end at Line 54.

Reply:
I took this approach from the Colemak file.
How does a `sendstring` file work?

Reply:
Oh, Colemak has a `sendstring` file, so I can mimic that, but will that render all the code related to "Make it easy to support these in macros" unnecessary?

Reply:
`sendstring_norman.h` would be the mapping for how to map ASCII keycodes to the physical keys used by Norman.

Notice that `sendstring_colemak.h`, for its alpha keys, uses `CM_*` instead of `KC_*`. The file is basically saying "To type an F, send the keycode associated with Colemak's location for F (`CM_F`)." [`CM_F` is mapped to `KC_E`](https://github.com/qmk/qmk_firmware/blob/22ba36a4d86c06ce1b6049555525646eb16bb57a/quantum/keymap_extras/keymap_colemak.h#L23) in `keymap_colemak.h`.

Reply:
Sorry about the lag - I think I've done it in the right way now.